### DISCUSSION TITLE
Arnold options in USD export?

### DISCUSSION THREAD

**Discussion by sharktacos** [Autodesk/maya-usd#3274] [Q&A] [answered]
Using Maya 2024.1 (USD 0.23)

The Arnold-USD export has the option to choose what exactly to export (and what not to). Render options, drivers, fliters, color management, etc.

<img width="321" alt="image" src="https://github.com/Autodesk/maya-usd/assets/23034637/18b2bcb3-75c6-4615-95e5-af08f1451851">

Is the same achievable with the Maya USD export?

**Comment by wallworm** [comment, marked as answer]
This is not currently available in the Maya USD plugin. However, we are looking into this ability right now.

**Comment by womanyoyoyo-adsk** [comment]
HI @sharktacos , we just added the ability to suppress meshes, cameras and lights on export in our 0.26.0 release.
